testkit/src/main/scala/org/bitcoins/testkit/node/NodeTestUtil.scala

```diff
@@ -29,16 +35,42 @@ abstract class NodeTestUtil extends P2PLogger {
     */
   def getBitcoindSocketAddress(
       bitcoindRpcClient: BitcoindRpcClient): InetSocketAddress = {
```

Comment:
Probalby should make this return a `Future[InetSocketAddress]` and remove the `Await.result()`.